manifests/hub/hub-clusterrole.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:open-cluster-management:{{ .HubCoreName }}
+rules:
+# Allow hub to monitor and update status of csr
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["certificatesigningrequests"]
+  verbs: ["create", "get", "list", "watch"]
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["certificatesigningrequests/status"]
+  verbs: ["update"]
+# Allow hub to get/list/watch/create/delete namespace and service account
+- apiGroups: [""]
+  resources: ["namespaces", "serviceaccounts", "configmaps", "events"]
+  verbs: ["get", "list", "watch", "create", "delete", "update"]
+# Allow hub to manage clusterrole/clusterrolebinding/role/rolebinding
```

Comment:
you'll eventually need the rbac permission required for expanding powers (granting permissions you don't have), but this can get you started.

Reply:
add the escalate verb.